frontend/src/styles/globals.css

```
@@ -8,10 +8,36 @@
+    --color-bg-base: #f9fafb;     /* gray-50 */
+}
+
+:root.dark {
+    color-scheme: dark;
+    --color-text-base: #f9fafb;   /* gray-50 */
+    --color-text-muted: #cbd5e1;  /* slate-300 — ~7:1 on gray-800/900 */
+    --color-bg-base: #111827;     /* gray-900 */
+}
+
 body {
     font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)